docs/fundamentals/networking/http/http-overview.md

```diff
@@ -0,0 +1,150 @@
+---
+title: HTTP support in .NET
+description: Learn about the comprehensive support for HTTP that .NET provides.
+ms.date: 08/24/2022
+helpviewer_keywords:
+  - "protocols, HTTP"
+  - "sending data, HTTP"
+  - "HttpWebResponse class, sending and receiving data"
+  - "HTTP"
+  - "receiving data, HTTP"
+  - "application protocols, HTTP"
+  - "Internet, HTTP"
+  - "network resources, HTTP"
+  - "HTTP, about HTTP"
+  - "HttpWebRequest class, sending and receiving data"
+---
+
+# HTTP support in .NET
+
+Hypertext Transfer Protocol (or HTTP) is a protocol for requesting resources from a web server. The <xref:System.Net.Http.HttpClient?displayProperty=fullName> class exposes the ability to send HTTP requests and receive HTTP responses from a resource identified by a URI. There are many types of resources that are available on the web, and HTTP defines a set of request methods for accessing these resources.
+
+## HTTP request methods
+
+The request methods are differentiated via several factors, first by their _verb_ but also by the following characteristics:
+
+- A request method is **_idempotent_** if it can be successfully processed multiple times without changing the result. For more information, see [RFC 7231: Section 4.2.2 Idempotent Methods](https://datatracker.ietf.org/doc/html/rfc7231#section-4.2.2).
+- A request method is **_cacheable_** when its corresponding response can be stored for reuse. For more information, see [RFC 7231: Section 4.2.3 Cacheable Methods](https://datatracker.ietf.org/doc/html/rfc7231#section-4.2.3).
+- A request method is considered a **_safe method_** if it doesn't modify the state of a resource. All _safe methods_ are also _idempotent_, but not all _idempotent_ methods are considered _safe_. For more information, see [RFC 7231: Section 4.2.1 Safe Methods](https://datatracker.ietf.org/doc/html/rfc7231#section-4.2.1).
+
+| HTTP verb | Is idempotent | Is cacheable         | Is safe |
+|-----------|---------------|----------------------|---------|
+| `GET`     | ✔️ Yes       | ✔️ Yes               | ✔️ Yes |
+| `POST`    | ❌ No         | ⚠️ <sup>†</sup>Rarely | ❌ No   |
+| `PUT`     | ✔️ Yes       | ❌ No                 | ❌ No   |
+| `PATCH`   | ❌ No         | ❌ No                 | ❌ No   |
+| `DELETE`  | ✔️ Yes       | ❌ No                 | ❌ No   |
+| `HEAD`    | ✔️ Yes       | ✔️ Yes               | ✔️ Yes |
+| `OPTIONS` | ✔️ Yes       | ❌ No                 | ✔️ Yes |
+| `TRACE`   | ✔️ Yes       | ❌ No                 | ✔️ Yes |
+| `CONNECT` | ❌ No         | ❌ No                 | ❌ No   |
+
+> <sup>†</sup>The `POST` method is only cacheable when the appropriate `Cache-Control` or `Expires` response headers are present. This is very uncommon in practice.
+
+## HTTP status codes
```

Comment:
Should we link to the standard here? https://www.rfc-editor.org/rfc/rfc9110#name-status-codes

Reply:
I'll add a link.